dll/shellext/fontext/CFontExt.hpp

```diff
@@ -81,4 +88,8 @@ class CFontExt :
         COM_INTERFACE_ENTRY_IID(IID_IPersistFolder, IPersistFolder)
```

Comment:
If `IDropTarget` should also be available via `QueryInterface`, add it in this table.

Reply:
OK.